docs/upgrade-guides/node-to-express.mdx

```diff
@@ -0,0 +1,205 @@
+---
+description: Learn how to upgrade from the Clerk's Node SDK to the Express SDK.
+---
+
+# Upgrade from Clerk's Node SDK to the Express SDK
+
+## Install the Express SDK
+
+To upgrade from the Clerk Node SDK to the Clerk Express SDK, uninstall the Node SDK by running the following command:
+
+<CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+  ```sh {{ filename: 'terminal' }}
+  npm uninstall @clerk/clerk-sdk-node
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  yarn remove @clerk/clerk-sdk-node
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  pnpm remove @clerk/clerk-sdk-node
+  ```
+</CodeBlockTabs>
+
+Then, install the Express SDK by running the following command:
+
+<CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+  ```sh {{ filename: 'terminal' }}
+  npm install @clerk/express
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  yarn add @clerk/express
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  pnpm add @clerk/express
+  ```
+</CodeBlockTabs>
+
+## Migrate from `withAuth()`
+
+If you previously used `withAuth()` to wrap your route handlers, you can now use [`getAuth()`](/docs/references/express/overview#get-auth) with the Express SDK. Pass the request object to `getAuth()` to retrieve the authentication state within the route handler.
+
+`clerkMiddleware()` is required to be set in the middleware chain before this util is used, as shown in the following example.
+
+```ts {{ del: [1, [10, 13]], ins: [2, 8, [14, 17]] }}
+import { withAuth } from '@clerk/clerk-sdk-node'
+import { clerkMiddleware, getAuth } from '@clerk/express'
+import express from 'express'
+
+const app = express()
+const port = 3000
+
+app.use(clerkMiddleware())
+
+app.get(
+  '/path',
+  withAuth((req, res) => res.json(req.auth)),
+)
+app.get('/auth-state', (req, res) => {
+  const authState = getAuth(req)
+  return res.json(authState)
+})
+
+app.listen(port, () => {
+  console.log(`Server is running on http://localhost:${port}`)
+})
+```
+
+## `requireAuth()` usage changes
+
+In the Node SDK, `requireAuth()` wrapped route handlers. With the Express SDK, you must update `requireAuth()` to function as middleware instead of wrapping your handlers.
```

Comment:
```suggestion
In the Node SDK, `requireAuth()` wrapped route handlers. With the Express SDK, you must instead pass `requireAuth` as a middleware function before your handler instead of wrapping your handler.
```